src/pages/home/sidebar/SidebarLinks.tsx

```diff
@@ -16,37 +15,33 @@ import KeyboardShortcut from '@libs/KeyboardShortcut';
 import Navigation from '@libs/Navigation/Navigation';
 import onyxSubscribe from '@libs/onyxSubscribe';
 import * as ReportActionContextMenu from '@pages/home/report/ContextMenu/ReportActionContextMenu';
-import safeAreaInsetPropTypes from '@pages/safeAreaInsetPropTypes';
 import * as App from '@userActions/App';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {Modal, Policy, Report} from '@src/types/onyx';
 
-const basePropTypes = {
-    /** Toggles the navigation menu open and closed */
-    onLinkClick: PropTypes.func.isRequired,
-
-    /** Safe area insets required for mobile devices margins */
-    insets: safeAreaInsetPropTypes.isRequired,
+type SidebarLinksOnyxProps = {
+    activePolicy: OnyxEntry<Policy>;
 };
 
-const propTypes = {
-    ...basePropTypes,
-
-    optionListItems: PropTypes.arrayOf(PropTypes.string).isRequired,
-
-    isLoading: PropTypes.bool.isRequired,
-
-    // eslint-disable-next-line react/require-default-props
-    priorityMode: PropTypes.oneOf(_.values(CONST.PRIORITY_MODE)),
-
-    isActiveReport: PropTypes.func.isRequired,
+type SidebarLinksProps = SidebarLinksOnyxProps & {
+    onLinkClick: () => void;
+    insets: EdgeInsets | undefined;
+    optionListItems: string[] | null;
```

Comment:
Why do you need `null`? It looks like we can remove it
```suggestion
    optionListItems: string[];
```

Reply:
If you look at `SidebarLinksData` component there we are passing `optionsListItems` which can be `string[] | null` so that why here we also need that type

Reply:
@kubabutkiewicz Yeah, but I've suggested some updates there as well
https://github.com/Expensify/App/pull/36847#discussion_r1502947777